pkgs/by-name/pr/proj-data/tests.nix

```diff
@@ -0,0 +1,21 @@
+{ runCommand, proj, proj-data }:
+
+let
+  inherit (proj-data) pname;
+in
+runCommand "${pname}-tests" { meta.timeout = 60; }
+  ''
+    # add proj-data files to proj resources search path
+    export PROJ_DATA=${proj}/share/proj:${proj-data}
```

Comment:
Should this possibly be set by the package itself?

Reply:
I am not sure what is the best approach. We can wrap all executables with `PROJ_DATA` path, but it wouldn't help with dozens of other apps using `proj` as library (like QGIS for example).

Reply:
> I am not sure what is the best approach. We can wrap all executables with `PROJ_DATA` path, […]

Sounds good.

> but it wouldn't help with dozens of other apps using `proj` as library (like QGIS for example).

Isn't that unrelated? We'd be making one thing work without breaking anything else, right?